.github/workflows/update-flagsmith-environment.yml

```diff
@@ -0,0 +1,41 @@
+name: Update Flagsmith Defaults
+
+on:
+  schedule:
+    - cron: 0 8 * * *
+
+defaults:
+  run:
+    working-directory: api
+
+jobs:
+  update_server_defaults:
+    runs-on: ubuntu-latest
+    name: Update API Flagsmith Defaults
+    env:
+      FLAGSMITH_API_URL: https://edge.api.flagsmith.com/api/v1
+      FLAGSMITH_SERVER_KEY: ${{ secrets.FLAGSMITH_SERVER_KEY }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+          cache: pip
+
+      - name: Install Dependencies
+        run: make install
+
+      - name: Update defaults
+        run: python manage.py updateflagsmithenvironment
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
+        with:
+          commit-message: Update API Flagsmith Defaults
+          branch: chore/update-api-flagsmith-environment
+          delete-branch: true
+          title: 'chore: update Flagsmith environment document'
+          labels: api
```

Comment:
Can we make this step conditional depending on whether the env actually changes or not? See [example](https://github.com/orgs/community/discussions/27148#discussioncomment-3254824).

Reply:
Awesome.